### DISCUSSION TITLE
unable to install any PHP version because failing to download aom--homebrew-bus_qcif_15fps.y4m

### DISCUSSION THREAD

**Discussion by michaelnguyen2021** [shivammathur/homebrew-php#1552] [Q&A / Help]
I got this error message

==> Fetching aom
==> Downloading https://media.xiph.org/video/derf/y4m/bus_qcif_15fps.y4m

curl: (60) SSL certificate problem: self signed certificate
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
Error: php@8.1: Failed to download resource "aom--homebrew-bus_qcif_15fps.y4m"
Download failed: https://media.xiph.org/video/derf/y4m/bus_qcif_15fps.y4m

It looks like the URL has an invalid SSL certificate. I can not install any shivammathur PHP versions atm. Any workarounds?

**Comment by shivammathur** [comment]
Please run `brew update` and then `brew doctor`. If it shows any errors, please fix them and then try again.
